sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -867,7 +883,15 @@ def _delete_dataset(self, project_id, dataset_id, delete_contents=True):
   @retry.with_exponential_backoff(
       num_retries=MAX_RETRIES,
       retry_filter=retry.retry_on_server_errors_and_timeout_filter)
-  def create_temporary_dataset(self, project_id):
+  def get_table_location(self, project_id, dataset_id, table_id):
```

Comment:
yes, it is used in line 617:

    source_location = self.client.get_table_location(
            source_project_id, source_dataset_id, source_table_id)

alternatively, we add a `client.get_table` function and get the `table.location` from there. There is already a `client.get_or_create_table` function, but I would refrain from using for getting only the `location`.